src/main/java/de/zalando/ep/zalenium/container/DockerContainerClient.java

```diff
@@ -139,8 +139,11 @@ private String getContainerId(String containerName) {
         }
 
         if (containerList != null) {
-	        return containerList.stream()
-	                .filter(container -> containerNameSearch.equalsIgnoreCase(container.names().get(0)))
+            String containerByName = containerList.stream()
+                    .filter(container -> containerNameSearch.equalsIgnoreCase(container.names().get(0)))
+                    .findFirst().map(Container::id).orElse(null);
+            return containerByName != null ? containerByName : containerList.stream()
+	                .filter(container -> container.names().get(0).contains(containerName))
```

Comment:
I only tested it in my machine and tests run fine, and zalenium creates and deletes containers without any issues, which is as expected because my changes shouldn't affect anything outside of AWS (unless the zalenium container has a different name than the given one). 

As for AWS I don't know yet, zalenium is up and I can see the initial selenium containers  (with the naming convention you mention), but I couldn't run any tests yet because for some reason the port in the AWS container that should be mapped to zalenium's port (4444) isn't open, but that's a problem with our infrastructure. 

If you think the best course of action would be to merge this after I make sure that tests can be run on AWS just to be sure it works then I don't mind.

Reply:
@diemol I solved that problem and ran a few tests, I can confirm that my fix works on AWS too

Reply:
@MarianoSalerno can you please share your fix/task definition too?
In my case, I tried putting name as environment variable in task definiton but it still comes up with <task-denifition>-Randomstring format.
Thanks in advance